src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,25 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Machine;
+import core.basesyntax.samples.Bulldozer;
+import core.basesyntax.samples.Excavator;
+import core.basesyntax.samples.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
```

Comment:
let's create a `Machine` array. After that you can go through this array, using for loop and call methods `doWork()` and `doStop()` in it.